src/core/qgsvectorfilewriter.cpp

```diff
@@ -962,6 +962,21 @@ class QgsVectorFileWriterMetadataContainer
                              )
                            );
 
+      // FlatGeobuf
```

Comment:
I missed it earlier, but this should be ifdefed in only for GDAL versions with support for the format...

Reply:
Alright, fixed. I see now that also the reading side (ogrprovider) should get some attention but I'll do that in a separate PR.